CSC322-Grinnell/mayflower-directory
# Fix image alignment in Pictures

Resolves #148 

# Changes

1. Moved image decision logic (originally in view for "pictures" and "show" page) into profile controller.
2. Changed pictures layout to flexbox and cleaned up its spacing

# Testing

1. Go to branch: pictures 
2. $ rake db:migrate 
3. $ rake db:seed
4: $ rake db:dummy_seed
5. $ rails s -b $IP -p $PORT
6. Preview the app and check the pictures page
7. Make sure all profiles pictures look good.
8. Review related code

#warning: this relies on 'dummy_seed'
